samples/v1/test/cloudscheduler.test.yaml

```diff
@@ -0,0 +1,18 @@
+type: test/samples
```

Comment:
How is this file authored, and maintained?  Guessing it exists in multiple repositories, one for each language, right?

Reply:
No, there's one test.yaml for all languages. It lives with the sample config and proto. SynthTool generates the samples but only copies the test config to GitHub.